components/MobileNav.tsx

```diff
@@ -20,122 +21,132 @@ import {
   NavbarLinkWhyUse
 } from "./NavbarComponents"
 
-const NavBarBoxContainer: FC<
-  React.PropsWithChildren<{ className?: string }>
-> = ({ children, className }) => {
-  return (
-    <div
-      className={`d-flex flex-row align-items-start justify-content-between w-100`}
-    >
-      {children}
-    </div>
-  )
-}
+export const MobileNav: React.FC<React.PropsWithChildren<unknown>> = () => {
+  const BlackCollapse = styled(() => {
+    return (
+      <Navbar.Collapse id="basic-navbar-nav" className="bg-black mt-2 ps-4">
+        {/* while MAPLE is trying to do away with inline styling,   *
+         *  both styled-components and bootstrap classes have been  *
+         *  ignoring height properties for some reason              */}
+        <div style={{ height: "100vh" }}>
+          {whichMenu == "site" ? <SiteLinks /> : <ProfileLinks />}
+        </div>
+      </Navbar.Collapse>
+    )
+  })`
+    .bg-black {
```

Comment:
nit: Could this just use the bootstrap provided `bg-dark` and save some boilerplate? IMO we should probably start pushing designs to standardize on our bootstrap theme colors anyway.